devnet/README.md

```diff
@@ -51,4 +51,25 @@ This key is injected in `alice.Dockerfile` so it uses the same public key for th
 
 ### docker-compose.yml
 
-The Docker Compose file.  Specifies the IP addresses of all the nodes.  
\ No newline at end of file
+The Docker Compose file.  Specifies the IP addresses of all the nodes. 
```

Comment:
:eyes: TWO SPACES SPOTTED!

```suggestion
The Docker Compose file. Specifies the IP addresses of all the nodes. 
```

Actually I see you use double spaces everywhere after full stops, why :thinking: I'm fine with it, just curious

Reply:
I have no idea actually. I don't do that while typing, maybe VS code is doing it for me for some reason.